apps/web/src/App.tsx

```diff
@@ -1,27 +1,13 @@
-import React from "react";
-import { Link } from "ui";
-import "./App.css";
+import React from 'react'
+import { Link } from 'ui'
+import './App.css'
 
-function App() {
+const App = () => {
```

Comment:
In TS you should also always type the return value of your functions as well as any parameters:
```ts
const returnsInput = (inputString : String) : String => inputString;
```
This convention extends to React functional components:
```suggestion
const App = () : React.ReactElement => {
```